fdbserver/DataDistribution.actor.cpp

```diff
@@ -481,17 +481,21 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 
 		for (; it != self->initData->dataMoveMap.ranges().end(); ++it) {
 			const DataMoveMetaData& meta = it.value()->meta;
+			if (meta.ranges.empty()) {
```

Comment:
I wonder if this show be a SevWarn event, or ASSERT()

Reply:
Done.